npm_test.go

```diff
@@ -31,7 +31,8 @@ import (
 
 type npmTestParams struct {
 	testName       string
-	command        string
+	nativeCommand  string
+	legacyCommand  string
```

Comment:
Let's add ```// Deprecated``` comment above this lne, so that we remember to remove it in the future. 
The same goes for the ```TestNpmLegacy``` function.